lib/srv/desktop/audit_cache.go

```diff
@@ -35,169 +34,110 @@ type readRequestInfo struct {
 
 type writeRequestInfo readRequestInfo
 
-const (
-	// entryMaxItems is the maximum number of items we want
-	// to allow in a single sharedDirectoryAuditCacheEntry.
-	//
-	// It's not a precise value, just one that should give us
-	// prevent the cache from growing too large due to a
-	// misbehaving client.
-	entryMaxItems = 2000
-)
-
-type sharedDirectoryAuditCacheEntry struct {
```

Comment:
We've removed one layer of indirection here. An audit cache is associated to a single session, so the top level map of session ID to entry no longer needs to exist.

(And all the method signatures have been updated to remove the session ID argument)